src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+* Update export logic for scopes:
+  * Export scopes which have a non-null key as individual columns (each
+    key-value pair from the scopes is exported as its own column; these columns
+    would also be taken into consideration when the CustomFields option is
+    applied)
+  * GenevaExporter would not check for duplicate keys (if the scopes and log
```

Comment:
I'd remove the part about duplicate. Its already the existing behavior, so don't think it should be in changelog.